src/plugins/forceOwnerCrown.ts

```diff
@@ -38,6 +38,11 @@ export default definePlugin({
         },
     ],
     isGuildOwner(props) {
+        // Check if channel is a Group DM, if so return false
+        if (props.channel.type === 3) {
```

Comment:
```suggestion
        if (props?.channel?.type === 3) {
```

I've also thought about checking the type. Are you sure `3` is only used for group channels?

Reply:
![image](https://user-images.githubusercontent.com/30734036/198898112-412fa645-97bc-41b8-a6a8-a763f3d9db77.png)
According to docs yes